airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalAttemptExecution.java

```diff
@@ -79,7 +98,50 @@ public T get() throws TemporalJobException {
       LOGGER.info("Executing worker wrapper. Airbyte version: {}", EnvConfigs.AIRBYTE_VERSION);
       jobRootDirCreator.accept(jobRoot);
 
-      return execution.apply(jobRoot);
+      final CompletableFuture<T> outputFuture = new CompletableFuture<>();
+
+      final Worker<INPUT, T> worker = workerSupplier.apply(jobRoot);
+
+      final Thread workerThread = new Thread(() -> {
```

Comment:
nit: I tend to adopt uncle bob's recommendations of writing functions that aren't too long. This method is straightforward so it's more of a stylistic thing here. Not blocking.